lib/rubocop/cop/lint/duplicated_key.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for duplicated keys in hashes
+      #
+      # This cop mirrors a warning in ruby 2.2.
+      #
+      # @example
+      #   hash = { food: 'apple', food: 'orange' }
+      class DuplicatedKey < Cop
+        MSG = 'Duplicated key in hash'
+
+        def on_hash(node)
+          keys = []
+
+          hash = *node
+          hash.each do |pair|
+            key = pair.children.first
+            if keys.include?(key) && key.type != :send
```

Comment:
I think you should limit this to only literal types (which includes collections made up of literals), as something like `[method, method, method]` will outwit the cop.